eapp/public/js/acquisitionform.js

```diff
@@ -60,13 +62,14 @@ function addAqFields(formName){
 /**
 Add fields to the acquistion form UI using a specified JSON file
 */
+var fieldsCorrect = true
 function add_term_to_form(termform){
   selectedFields = termform['fields']
   console.log("Number of Fields in the form: ",selectedFields.length)
   //console.log("x",x.length)
 
   for (let i=0; i<selectedFields.length; i++){
-    let sid = "ndar-"+i
+    var sid = "ndar-"+i
```

Comment:
does using 'var' instead of 'let' resolves scoping issue? As we discussed, we probably want to use form key as id to the corresponding HTML element. In a different pull request, you may refactor to change ids from (ndar-i ) to selectedFields[i].id